### ISSUE TITLE
Button for speech speed has no accessible name

### ISSUE THREAD

**Issue by jimmydarko** [edrlab/thorium-reader#2710]
The button for adjusting the speech speed has no accessible name. I use a screen reader (JAWS) because I am visually impaired.
